client/app/scripts/liveblog-edit/module.js

```diff
@@ -480,18 +480,17 @@ define([
         function canPublishAPost(blog) {
             return privileges.userHasPrivileges({'publish_post': 1});
         }
-        function isUserOwner(archive) {
-            if ($rootScope.currentUser._id === archive.original_creator || $rootScope.currentUser.user_type === 'administrator') {
-                return true;
-            } else {
-                return false;
-            }
+        function isAdmin() {
+            return $rootScope.currentUser.user_type === 'administrator';
+        }
+        function isUserOwnerOrAdmin(archive) {
+            return $rootScope.currentUser._id === archive.original_creator || isAdmin();
         }
         function goToSettings() {
             var def = $q.defer();
             blogService.get($route.current.params._id)
             .then(function(response) {
-                if (isUserOwner(response)) {
+                if (isUserOwnerOrAdmin(response)) {
```

Comment:
I added 'orAdmin' to reflect more precisely what this function test. The name can't change any more since the name match now perfectly with what the function does.